config.toml

```diff
@@ -6,7 +6,7 @@ enableRobotsTXT = true
 # Hugo allows theme composition (and inheritance). The precedence is from left to right.
 theme = ["docsy"]
 
-timeout = 50000 # uh oh
+timeout = 500000 # uh oh
```

Comment:
In welcher Konfiguration sind 500 Sekunden Timeout notwendig?

Reply:
Das wurde notwendig beim erstmaligen Aufsetzen des lokalen Repository. Da ich anfangs ständig in ein Timeout gelaufen bin hat mir ein Fachmann vom makerspace das empfohlen.

Reply:
Das ist schon okay so. 50 sekunden war schon viel, 500 ist halt einfach noch mehr.

Hugo skaliert die Bilder immer schön runter, das dauert je nach Menge der Bilder und Geschwindigkeit des Computers beim ersten Mal sehr lang. Ab da ist es im Cache und geht ziemlich schnell.